ui/src/assets/i18n/fr.json

```diff
@@ -358,6 +358,8 @@
   "variable_updated" : "Variable mise à jour",
   "variable_value" : "Valeur",
 
+  "ui_updated" : "L'interface vient d'être mise à jour. Merci de cliquer dans cette zone pour rafraichir votre page.",
```

Comment:
french reform of 1990 says that you should forget ^ on i and u :)
http://www.academie-francaise.fr/sites/academie-francaise.fr/files/rectifications_1990.pdf